### ISSUE TITLE
HELP HOW DO I EXIT VIM PLEASE

### ISSUE THREAD

**Issue by hKQwHW** [hakluke/how-to-exit-vim#245]
HELP I PUT THE NUCLEAR CODES IN VIM BUT I CANT EXIT VIM I NEED TO GIVE THE NUKE CODES QUICKLY ASAP HELP EXIT VIM THX

**Comment by Adamekka** [comment]
:q!

**Comment by ssterling** [comment]
Emoticons aren’t appropriate at such a _critical_ moment like this, Adamekka.

**Comment by c3d** [comment]
I don't think you can. You should have used Emacs, where it's simply Control-X Control-C (you may need to confirm if some off the processes associated with the nuke codes are still active).
